core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -95,6 +94,30 @@ export class DotEditContentFormComponent implements OnInit {
         });
     }
 
+    /**
+     /**
+     * Emits the form value through the `formSubmit` event.
+     *
+     * @param {*} value
+     * @memberof DotEditContentFormComponent
+     */
+    onFormChange(value) {
+        this.formData.contentType.fields.forEach(({ variable, fieldType }) => {
+            // Shorthand for conditional assignment
+
+            if (FLATTENED_FIELD_TYPES.includes(fieldType as FIELD_TYPES)) {
+                value[variable] = value[variable]?.join(',');
+            }
+
+            if (CALENDAR_FIELD_TYPES.includes(fieldType as FIELD_TYPES)) {
+                value[variable] = value[variable]
+                    ?.toISOString()
+                    .replace(/T|\.\d{3}Z/g, (match: string) => (match === 'T' ? ' ' : '')); // To remove the T and .000Z from the date)
+            }
```

Comment:
Should we consider creating a handler function for this?

```js
value[variable] = valueFormatter(value);
```

Reply:
We create one [core-web/libs/edit-content/src/lib/components/dot-edit-content-form/utils.ts](https://github.com/dotCMS/core/blob/175069cdfba84491b4ceea42c2c5e150a85bf07b/core-web/libs/edit-content/src/lib/components/dot-edit-content-form/utils.ts) here